packages/ui/src/components/PinPad.tsx

```diff
@@ -71,15 +77,16 @@ const PinNumber = ({ character, onPressPinNumber, disabled }: PinNumberProps) =>
 
 export interface PinPadProps {
   onPressPinNumber: (character: PinValues) => void
+  useBiometricsPad?: boolean
   disabled?: boolean
 }
 
-export const PinPad = ({ onPressPinNumber, disabled }: PinPadProps) => {
+export const PinPad = ({ onPressPinNumber, useBiometricsPad, disabled }: PinPadProps) => {
   const pinValues = [
     [PinValues.One, PinValues.Two, PinValues.Three],
     [PinValues.Four, PinValues.Five, PinValues.Six],
     [PinValues.Seven, PinValues.Eight, PinValues.Nine],
-    [PinValues.Empty, PinValues.Zero, PinValues.Backspace],
+    [useBiometricsPad ? PinValues.Biometrics : PinValues.Empty, PinValues.Zero, PinValues.Backspace],
```

Comment:
there is https://oblador.github.io/react-native-keychain/docs/api/functions/getSupportedBiometryType.

Already used here: https://github.com/animo/paradym-wallet/blob/56e64ac227046c6b6d8906911ef0cdcf828182bc/packages/secure-store/secure-wallet-key/walletKeyStore.ts#L65